_posts/en/newsletters/2022-08-10-newsletter.md

```diff
@@ -0,0 +1,184 @@
+Bitcoin Core PR Review Club, announcements of new releases and release
+candidates, and descriptions of notable changes to popular Bitcoin
+infrastructure projects.
+
+## News
+
+- **Lowering the default minimum transaction relay feerate:** Bitcoin
+  Core only relays individual unconfirmed transactions that pay a
+  [feerate of at least one satoshi per vbyte][topic default minimum
+  transaction relay feerates] (1 sat/vbyte).  If a node's mempool fills
+  with transactions paying at least 1 sat/vbyte, then a higher feerate
+  will need to be paid.  Transactions paying a lower feerate can still
+  be included in blocks by miners and those blocks will be relayed.
+  Other node software implements similar policies.
+
+    Lowering the default minimum feerate has been discussed in the past
+    (see [Newsletter #3][news3 min]) but [hasn't been merged][bitcoin
+    core #13922] into Bitcoin Core.  The topic saw renewed
+    [discussion][chauhan min] in the past couple weeks:
+
+    - *Individual change effectiveness:* it was [debated][todd min] by
+      [several][vjudeu min] people how effective it was for individual
+      node operators to change their policies.
+
+    - *Past failures:* it was [mentioned][harding min] that the previous
+      attempt to lower the default feerate was hampered by the lower
+      rate also reducing the cost of several minor denial-of-service
+      (DoS) attacks.
+
+    - *Alternative relay criteria:* it was [suggested][todd min2] that
+      transactions violating certain default criteria (such as the
+      default minimum feerate) could instead fulfill some separate
+      criteria that make DoS attacks costly---for example, if a modest amount
+      of hashcash-style proof of work committed to the transaction to
+      relay.
+
+    The discussion did not reach a clear conclusion as of this writing.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Decouple validation cache initialization from ArgsManager][review club 25527]
+is a PR by Carl Dong that separates node configuration logic from the
+initialization of signature and script caches.
+It is part of the [libbitcoinkernel project][].
+
+
+{% include functions/details-list.md
+  q0="In your own words, what does the `ArgsManager` do?
+Why or why not should `ArgsManager` belong in `src/kernel` versus `src/node`?"
+  a0="It's a global data structure for handling configuration options
+(`.bitcoin/bitcoin.conf` and command line arguments).
+It lets users customise the configuration of their nodes.
+It's undesirable for the consensus engine (`src/kernel`) to access `ArgsManager` for
+any reason, because it could, in principle, allow per-node configuration options
+to modify the behavior of the consensus engine. This could lead to
```

Comment:
@glozow:
> The nodes don't need to all have the same cache size

That's exactly what I initially thought, as I said [here](https://bitcoincore.reviews/25527#l-50):

> "validation caches ... Why belong in src/kernel?" I'm unclear on that. A "cache", in normal usage, means just a performance optimization (the more stuff you cache, the quicker you can get to it) ... so why are those caches related to consensus?

Then sipa [replied](https://bitcoincore.reviews/25527#l-52):

> If the cache has a bug, it may impact consensus.